packages/algob/sample-project/test/sample-test.js

```diff
@@ -0,0 +1,72 @@
+import { getProgram } from '@algorand-builder/algob';
+import { Runtime, StoreAccount, types } from '@algorand-builder/runtime';
+const { assert } = require('chai');
```

Comment:
Let's be consistent with the imports. Please change lines 1-2 to `const {..} = require('..')` OR change line 3 to `import { assert} from "chai"`